src/components/transactions/Repay/RepayModalContent.tsx

```diff
@@ -153,6 +154,11 @@ export const RepayModalContent = ({
     .multipliedBy(marketReferencePriceInUsd)
     .shiftedBy(-USD_DECIMALS);
 
+  const maxRepayWithDustRemaining =
+    isMaxSelected &&
+    displayAmountAfterRepayInUsd.toNumber() > 0 &&
+    displayAmountAfterRepayInUsd.toNumber() < 5;
```

Comment:
Why the lower 5?

Reply:
The tooltip message seems odd unless you're actually leaving a small amount behind. Like if I have 10000 DAI debt and repay my max wallet balance of 50 DAI, having a warning for leaving dust doesn't seem relevant

I can remove it though if you think otherwise

Reply:
I would remove it as the 5 is somewhat arbitrary (5 one tokens are basically dust). It doesn't hurt to show it imo.